tests/functional/events/11-cycle-task-event-job-logs-retrieve/flow.cylc

```diff
@@ -1,3 +1,4 @@
+#!jinja2
```

Comment:
Why?

This config doesn't use Jinja2!

Reply:
But template variables are supplied - looks like there's vestigial garbage in the test.